src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/FooterBanner.tsx

```diff
@@ -1,24 +1,19 @@
-import { RouterLink } from "System/Router/RouterLink"
+import { Box, Text } from "@artsy/palette"
 import { FullBleedBanner } from "Components/FullBleedBanner"
-import { useWindowSize } from "Utils/Hooks/useWindowSize"
-import { breakpoints, Text } from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
 
 export const FooterBanner: React.FC = () => {
-  const { width } = useWindowSize()
-
   return (
-    <>
-      {width > parseInt(breakpoints.sm, 10) && (
-        <FullBleedBanner dismissable={false} variant="defaultLight" mb={-4}>
-          <Text variant="xs">
-            Gallery or art dealer?{" "}
-            <RouterLink to="https://partners.artsy.net">
-              Become a partner
-            </RouterLink>{" "}
-            to access the world’s largest online marketplace.
-          </Text>
-        </FullBleedBanner>
-      )}
-    </>
+    <Box display={["none", "block"]}>
+      <FullBleedBanner dismissable={false} variant="defaultLight" mb={-4}>
+        <Text variant="xs">
```

Comment:
You can also remove this `Text` wrapper.

Reply:
(It's already there: https://github.com/artsy/palette/blob/2f94a86cf74ba0226616d0474224245bb829eec2/packages/palette/src/elements/Banner/Banner.tsx#L32)

Reply:
on it, thanks @dzucconi